CHANGELOG.md

```diff
@@ -321,7 +322,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Support for stm32f407 and stm32f429.
 
-[Unreleased]: https://github.com/stm32-rs/stm32f4xx-hal/compare/v0.9.0...HEAD
+[unreleased]: https://github.com/stm32-rs/stm32f4xx-hal/compare/v0.9.0...HEAD
```

Comment:
That's most certainly wrong. The link above uses `Unreleased`.

Reply:
Hmm i didn't change that line manually, must have been some auto formatting going on 🤷🏻‍♀️

Reply:
Care to fix it so I can merge? 😅